scene-recognition/process.py

```diff
@@ -0,0 +1,67 @@
+# process script to copy raw csv into new golds directory
+# iterate through each raw directory (use yy-mm-dd pattern to find directories)
+# copy each csv file to golds directory
+# e.g for subdirectory(matches date format) in this workspace:
+#       for csv file in subdirectory:
+#           add file to scene-recognition/golds
+
+import os
+import re
+import shutil
+import pandas as pd
+
+def truncate_total(value):
+    """
+    This method takes in strings of filenames from the original format (filename column)
+    and truncates to just the total_ms value.
+    """
+    truncated = value.split("_")[-2]
+    return truncated
+def truncate_convert_ISO(value):
+    """
+    This method takes in strings of filenames from the original format (filename column)
+    and truncates it to just the millisecond value, and then converts the milliseconds to
+    ISO standard timestamps.
+    """
+    # get last portion of value, delimited by underscores
+    truncated = value.split("_")[-1]
+    # remove file extension
+    truncated = truncated[:-4]
+    ms = int(truncated)
+    # 3600000 milliseconds per hour, 60000 milliseconds per minute, 1000 miliseconds per second
+    hours = ms // 3600000
+    ms %= 3600000
+    minutes = ms // 60000
+    ms %= 60000
+    seconds = ms // 1000
+    ms %= 1000
+    timestamp = str(hours).zfill(2) + ":" + str(minutes).zfill(2) + ":" + str(seconds).zfill(2) + "." + str(ms).zfill(3)
+    return timestamp
+
+
+path = os.getcwd()
+folder = os.fsencode(path)
+goldpath = path + '\\golds'
```

Comment:
Code chocked here, since the handling of directory separator is different on my os (linux). Suggesting some alternatives; 

- https://docs.python.org/3/library/pathlib.html 
- https://docs.python.org/3/library/os.path.html#os.path.join
- https://docs.python.org/3/library/os.html#os.sep